lib/util/sdp/issue8329.js

```diff
@@ -87,6 +88,43 @@ function mediaSectionWorkaround(mediaSection) {
   return mediaSection;
 }
 
+/**
+ * @param {string} mediaSection
+ * @param {Map<PT, Codec>} ptToCodecName
+ * @returns @param {string} newMediaSection
+ */
+function deleteDuplicateRtxPts(mediaSection, ptToCodecName) {
+  // NOTE(syerrapragada): In some cases Chrome produces an offer/answer
+  // with duplicate "rtx" payload mapping in media section. When applied,
+  // Chrome rejects the SDP. We work around this by deleting duplicate
+  // "rtx" mappings found in SDP.
+  Array.from(ptToCodecName.keys()).forEach(pt => {
+    // 1. Find if there are duplicate PTs
+    const rtpmapPattern = new RegExp(`a=rtpmap:${pt}.*`, 'g');
+    const matches = mediaSection.match(rtpmapPattern);
+    if (matches && matches.length > 1) {
+      // 2. If orig is not 'rtx' type, delete all duplicate 'rtx' pts
```

Comment:
@syerrapragada 

If we generalize to delete any duplicate mapping, then we just need `3.` and we can get rid of `.2`. WDYT?